lang/de.json

```diff
@@ -826,12 +826,13 @@
       "windling": "Windling"
     },
     "Notifications": {
-      "Error": {},
-      "Info": {
+      "error": {},
```

Comment:
Why are these not capitalized anymore? I thought the rule is that only actual keys (those that have a translation) are written with lower letters?

Reply:
because, I missed this. probably just added them as small since the ui.notification.warn is small two and i didn't bothered at that moment to make a function to lowercase it